LIFXMasterApp.groovy

```diff
@@ -1856,6 +1856,19 @@ private Map deviceVersion(Map device) {
                             chain            : false
                     ]
             ]
+        case 57:
+        case 68:
+			return [
+					name      : 'LIFX Candle',
+					deviceName: 'LIFX Color',
+					features  : [
+							color            : true,
+							infrared         : false,
+							multizone        : false,
+							temperature_range: [min: 2500, max: 9000],
```

Comment:
we don't currently use this value -- but for any future support which might: LIFX lists these with a range 1500-9000

Reply:
Confirmed 1500K ; updated to be correct.  Also fixed some formatting introduced when I switched editors.